compiler/one-cmds/tests/one-build_neg_003.test

```diff
@@ -21,7 +21,7 @@ filename="${filename_ext%.*}"
 
 trap_err_onexit()
 {
-  if grep -q "'one-build' section is required in configuraion file" "${filename}.log"; then
+  if grep -q "section is required in configuraion file" "${filename}.log"; then
```

Comment:
using `[]` seems to work as regex pattern match and `[section]` fails to work

Reply:
Maybe `\[one-build\]`? Or is it backslash that makes error?

Reply:
> Maybe `\[one-build\]` ?

I'll try this one

> Or is it backslash that makes error?

Yes `[]` acts as regex pattern so `[one-build]` tries something and fails.

Reply:
`\[one-build\]` this works :)